src/test/java/com/booster/server/NoteControllerTest.java

```diff
@@ -46,49 +47,50 @@ void createNewNote() throws Exception {
 
         mockMvc.perform(MockMvcRequestBuilders.post("/note")
                         .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .content(objectMapper.writeValueAsString(new CreateNoteInput().setContent(firstNote))))
+                        .content(objectMapper.writeValueAsString(new CreateNoteInput().setContent(firstNoteContent))))
                 .andDo(print())
                 .andExpect(status().isCreated())
-                .andExpect(jsonPath("$.content").value(firstNote));
+                .andExpect(jsonPath("$.content").value(firstNoteContent));
 
         assertThat(noteRepository.findAll())
                 .hasSize(1)
                 .extracting("content")
-                .containsExactly(firstNote);
+                .containsExactly(firstNoteContent);
+        assertTrue(noteRepository.findAll()
+                .stream()
+                .allMatch(n -> n.getLastSeenAt().equals(LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIN))));
     }
 
     @Test
-    void correctUpdateOfLastSeenAtPropertyOfVocabularyEntry() throws Exception {
+    void correctUpdateOfLastSeenAtPropertyOfNote() throws Exception {
```

Comment:
What do you think if we change `correctUpdateOfLastSeenAtPropertyOfNote` to `correctNoteLastSeenAtUpdate`? Does it not sound more fluent?